pitest-maven-verification/pom.xml

```diff
@@ -32,13 +32,6 @@
 			<plugin>
 				<artifactId>maven-failsafe-plugin</artifactId>
 				<version>2.12</version>
-				<dependencies>
-					<dependency>
-						<groupId>org.apache.maven.surefire</groupId>
-						<artifactId>surefire-junit47</artifactId>
-						<version>2.12</version>
```

Comment:
I wonder why it was removed? surefire-junit47 became a default dependency in the current surefire (I could miss it as I  migrated to Gradle some time ago) or no features from surefire-junit47 are actually used in PIT?